hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -196,7 +194,7 @@ public class OmMetadataManagerImpl implements OMMetadataManager,
    * |----------------------------------------------------------------------|
    * |  snapshotInfoTable | /volume/bucket/snapshotName -> SnapshotInfo     |
    * |----------------------------------------------------------------------|
-   * |  renamedKeyTable | /volumeName/bucketName/objectID -> OmKeyRenameInfo|
+   * |  renamedKeyTable | /volumeName/bucketName/objectID -> renamedKey     |
```

Comment:
Let's rename the table to be more specific.

```suggestion
   * |  snapshotRenamedKeyTable | /volName/buckName/objID -> /v/b/origKey   |
```